gems/pending/util/miq_apache/miq_apache.rb

```diff
@@ -206,7 +206,7 @@ def self.create_balancer_config(opts = {})
     def self.create_redirects_config(opts = {})
       opts[:redirects].to_miq_a.each_with_object("") do |redirect, content|
         if redirect == "/"
-          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts)) /self_service/index.html [L]\n"
+          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts|bower_components|gettext)) /self_service/index.html [L]\n"
```

Comment:
Note, this is getting really complicated... we should probably try to understand if we can fix this in code via clearer routes so we don't need such a complicated rewrite rule.  For example, why are many of these redirecting but not all?